### PR TITLE
Update language names and region data

### PR DESCRIPTION
**Pull request by winstonsung**
* `cdo` Mindong
* `cjy` Jin
* `cpx` Pu–Xian Min
* `gan` Gan
* `nan` Minnan
* `sh` Serbo-Croatian
* `wuu` Wu
* `zh` Mandarin (Chinese)

* `CN` Mainland China
* `HK` Hong Kong
* `MY` Malaysia
* `SG` Singapore
* `TW` Taiwan

Bugs: 
* #253
* #314
* T354937 ( https://phabricator.wikimedia.org/T354937 )

Change-Id: I61befc6c1816a18899bb63d50f4aa29756d70605

### CODE REVIEW

File: data/langdb.yaml [hunk 1]
```diff
@@ -770,14 +770,14 @@ languages:
...
   wo: [Latn, [AF], Wolof]
-  wuu: [Hans, [AS], 吴语]
+  wuu: [Hani, [AS], 吴语]
   wuu-hans: [Hans, [AS], 吴语（简体）]
   wuu-hant: [Hant, [AS], 吳語（正體）]
   xal: [Cyrl, [EU], хальмг]
   xh: [Latn, [AF], isiXhosa]
   xmf: [Geor, [EU], მარგალური]
   xmm: [Latn, [AS, PA], Manado]
-  xsy: [Latn, [AS], SaiSiyat]
```
Comment (amire80) [inline data/langdb.yaml:780]: AFAIK, CamelCase is not a mistake. It's used in Ethnologue, for example. There may be a real reason for it.

File: data/langdb.yaml [hunk 2]
```diff
@@ -791,19 +791,19 @@ languages:
   za: [Latn, [AS], Vahcuengh]
   zea: [Latn, [EU], Zeêuws]
   zgh: [Tfng, [AF], ⵜⴰⵎⴰⵣⵉⵖⵜ ⵜⴰⵏⴰⵡⴰⵢⵜ]
- # world? (may apply to many varieties of Chinese)
```
Comment (amire80) [inline data/langdb.yaml:794]: The comment can be simply removed. I probably added it many years ago, thinking out loud. It's OK to put it in WW.

File: data/langdb.yaml (hunk 1 above)
Comment (WinstonSung) [reply data/langdb.yaml:780]: I would rather believe it's a mistake:

* https://www.cip.gov.tw/en/tribe/grid-list/733B28E29AD8C492D0636733C6861689/info.html?cumid=D0636733C6861689

File: data/langdb.yaml (hunk 2 above)
Comment (WinstonSung) [reply data/langdb.yaml:794]: Done.

File: data/langdb.yaml [hunk 3]
```diff
@@ -131,9 +131,9 @@ languages:
   cbk-zam: [cbk]
   ccp: [Cakm, [AS], 𑄌𑄋𑄴𑄟𑄳𑄦]
  # FIXME cdo will be modified later to split to Latin and Chinese more cleanly
-  cdo: [Latn, [AS], Mìng-dĕ̤ng-ngṳ̄]
+  cdo: [Latn, [AS], 閩東語 / Mìng-dĕ̤ng-ngṳ̄]
   cdo-hani: [Hani, [AS], 閩東語（漢字）]
-  cdo-latn: [Latn, [AS], Mìng-dĕ̤ng-ngṳ̄ Bàng-uâ-cê]
```
Comment (WinstonSung) [inline data/langdb.yaml:136]: Re-confirmed and seems to be intended. Going to revert.

Comment (WinstonSung) [reply data/langdb.yaml:136]: Done.

File: data/langdb.yaml (hunk 1 above)
Comment (WinstonSung) [reply data/langdb.yaml:780]: ^ @amire80

**Comment by WinstonSung** [comment]
CC:
* @amire80 
* @Nikerabbit

File: data/langdb.yaml (hunk 1 above)
Comment (WinstonSung) [reply data/langdb.yaml]: Resolved: Seems to be intended (the capitalization seems to represent different sounds)

> Looking at this book ( https://alilin.apc.gov.tw/files/ebook/16781928765b6a822c61063/ ), it seems "s" and "S" represent different sounds ( page with a table of sounds: https://alilin.apc.gov.tw/files/ebook/16781928765b6a822c61063/HTML5/pc.html#/page/38 )  and here ( https://alilin.apc.gov.tw/files/ebook/16781928765b6a822c61063/HTML5/pc.html#/page/254 ) for "本族的自稱用語" it says "SaySiyat"
> -- https://t.me/translatewiki/15328

Comment (WinstonSung) [reply data/langdb.yaml]: (reverted)

**Comment by winstonsung** [comment]
CC:
* @amire80 
* @Nikerabbit

**Comment by winstonsung** [comment]
https://phabricator.wikimedia.org/T354937 is not included in the pull request commit.

**Comment by winstonsung** [comment]
(rebased)

**Review by amire80 — APPROVED** [review]

**Comment by winstonsung** [comment]
> This workflow requires approval from a maintainer

**Comment by winstonsung** [comment]
> https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

**Pull request merged**